src/Transport/Topology/NumberOfTopicsInBundleCheck.cs

```diff
@@ -26,12 +30,17 @@ static class NumberOfTopicsInBundleCheck
                 {
                     var filter = $"startswith(path, '{bundlePrefix}') eq true";
                     var foundTopics = await namespaceManagerThatCanQueryAndFilterTopics.GetTopics(filter).ConfigureAwait(false);
-                    numberOfTopics = foundTopics.Count();
+                    numberOfTopics = CountTopicsInBundle(topicInBundleNameRegex, foundTopics);
                 }
                 namespaceBundleConfigurations.Add(namespaceConfiguration.Alias, numberOfTopics);
             }
 
             return namespaceBundleConfigurations;
         }
+
+        public static int CountTopicsInBundle(Regex topicInBundleNameRegex, IEnumerable<TopicDescription> topics)
```

Comment:
Why public? On develop this is [inlined](https://github.com/Particular/NServiceBus.AzureServiceBus/blob/develop/src/Transport/Topology/NumberOfTopicsInBundleCheck.cs#L24)

Reply:
Have I mention merging conflicts is awesome? 😃 
Will change.

Reply:
👍